src/Sentry.Android.AssemblyReader/ELFSharp/MachO/Dylib.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.IO;
+using System.Text;
+using ELFSharp.Utilities;
+
+namespace ELFSharp.MachO
+{
+    internal abstract class Dylib : Command
```

Comment:
do we need this? this is a mac thing

Reply:
Possibly not - I didn't make any attempt to trim off unused code from ELFSharp (it's had the minimal changes required to get our project to build with it). 

We could try to shrink it down. The linker probably removes unused code paths when publishing on Android but I guess we could slightly improve linker times by removing the code altogether.

Worth it?